examples/layer-browser/src/app.js

```diff
@@ -68,22 +70,30 @@ class App extends PureComponent {
         pitch: 0,
         bearing: 0
       },
+      orbitViewState: {
+        lookAt: [0, 0, 0],
+        distance: 3,
+        rotationX: -30,
+        rotationOrbit: 30,
+        orbitAxis: 'Y',
+        fov: 50,
+        minDistance: 1,
+        maxDistance: 20
+      },
       activeExamples: {
         ScatterplotLayer: true
       },
       settings: {
+        infovis: false,
         multiview: false,
         useDevicePixels: true,
         pickingRadius: 0,
         drawPickingColors: false,
 
-        separation: 0
-        // the rotation controls works only for layers in
-        // meter offset projection mode. They are commented out
-        // here since layer browser currently only have one layer
-        // in this mode, and that layer's data is rotation symmetrical
-        // rotationZ: 0,
-        // rotationX: 0
+        // Model matrix manipulation
+        separation: 0,
+        rotationZ: 0,
```

Comment:
rotationY ? cause the orbitAxis is 'Y'

Reply:
> rotationY ? cause the orbitAxis is 'Y'

Ah... that control is for geospatial layers. Will consider how to make it work for infovis.

Reply:
> rotationY ? cause the orbitAxis is 'Y'

Support for rotating infovis layers using model matrix is a relatively minor feature. Would be a different PR.